keyboards/handwired/aball/info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "keyboard_name": "aball",
+    "url": "https://github.com/brickbots/aball/",
+    "maintainer": "@brickbots",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0xC2F0",
+        "device_ver": "0x0001"
+    }
```

Comment:
Missing layouts.

Reply:
Yes...indeed.... but it has no buttons :-)  I can put a null there to indicate so?

Reply:
Not really, but take a look at the ploopyco/trackball_nano to see how I handled the lack of any keys.

Reply:
You have a `LAYOUT` macro with two buttons in aball.h.

Reply:
Aha, yes, thank you @fauxpark!  When I set this up I was not sure how to handle a matrix-less device and get it to compile.  I've now emulated what was done with the Ploopy nano and I hope it's a lot clearer and will pass the linter now as well.  Thanks for your patience!